REST_API_Development/PictogramFiles.md

```diff
@@ -0,0 +1,35 @@
+# Pictogram files
+
+As of May of 2019, the `web-api` has no pictograms stored in the repository. Some pictograms have been stored in the [DBInitializer](https://github.com/aau-giraf/web-api/blob/develop/GirafRest/Data/DBInitializer.cs) class but have since been removed, because storing images in a database is generally considered harmful. 
+
+As there no longer are any pictograms stored in the repository, it can be difficult to run a local version of the `web-api`. We have therefore created a script which generates sample pictograms. This script is found in `/scripts/pictogen.py`. The point of this script is to generate lots of pictograms and put them in the location where the `web-api` would normally look for them.
```

Comment:
```suggestion
As there no longer are any pictograms stored in the repository, it can be difficult to run a local version of the `web-api`. We have therefore created a script which generates sample pictograms. This script is found in [/scripts/pictogen.py](https://github.com/aau-giraf/web-api/blob/develop/scripts/pictogen.py). The point of this script is to generate lots of pictograms and put them in the location where the `web-api` would normally look for them.
```